src/templater.rb

```diff
@@ -119,6 +119,12 @@ def initialize(msg="Input hash may contain infinitely recursive ERB")
     def replace_erb(template, template_parameters)
       parameters_binding = template_parameters.instance_eval {binding}
       ERB.new(template).result(parameters_binding)
+    rescue NoMethodError => e
+      # May be useful to include the name of the unset parameter in this error,
+      # however this is tricky as by the time we attempt to access a method on
+      # it the unset parameter is just `nil` as far as we can see here.
```

Comment:
Leave it as the TODO for now as it adds the functionality. Will create a card in trello so we can come back to this.